### DISCUSSION TITLE
RTL Support

### DISCUSSION THREAD

**Discussion by natainditama** [mantinedev/mantine#3318] [Feature requests]
Hello,

**Is your suggestion related to a problem? Please describe.**
As you may know the Arabic/Persian alphabet are in reverse format compared with English , I mean if you write things from left to right ... in Persian you must write from right to left

 **Describe the solution you'd like**
 you could add RTL (right to left ) support

 Thanks , It would be great

**Comment by rtivital** [comment]
https://mantine.dev/guides/rtl/

**Comment by natainditama** [comment]
Thank you
